ccmbioinfo/lfs
# LFS-1142: Create a utility program for changing the version of the CARDS Docker image used in a docker-compose.yml file

This PR introduces the `upgrade_cards_image.py` utility program which changes the version of the CARDS Docker image that is specified in a `docker-compose.yml` file.

To test:
- `cd compose-cluster`
- `python3 generate_compose_yaml.py --oak_filesystem`
- `cp docker-compose.yml docker-compose.yml.old`
- `python3 upgrade_cards_image.py --cards_docker_tag 1.0`
- `diff docker-compose.yml.old docker-compose.yml`
- The only change displayed by the `diff` command should be the change of the CARDS Docker image version tag
